src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -452,6 +452,7 @@ static Uri GetDiagnosticUri(Document document, DiagnosticData diagnosticData)
 
             static LSP.Diagnostic ConvertToLspDiagnostic(DiagnosticData diagnosticData, SourceText text)
             {
+                Contract.ThrowIfNull(diagnosticData.Message);
```

Comment:
Are we sure we won't get these? When can message _be_ null? Should we be filtering them or doing a ?? "" here instead?

Reply:
Thanks. I think this can removed, as I rebased on Cyrus' PR and he handled this differntly, below on line 469.